.github/ISSUE_TEMPLATE/runtime-upgrade---upgrade-title-version-.md

```diff
@@ -0,0 +1,52 @@
+---
+name: 'Runtime Upgrade: [Upgrade Title/Version]'
+about: Checklist for tracking the progress of a runtime upgrade on the Paseo Testnet.
+title: ''
+labels: runtime-upgrade
+assignees: ''
+
+---
+
+# Paseo Testnet Runtime Upgrade Checklist
+
+## Preparation
+
+- [ ] **Upgrade Proposal Review**: Confirm that the upgrade proposal has been fully reviewed and approved.
+- [ ] **Impact Analysis Completed**: Assess and document the expected impact of the upgrade on existing systems and users.
+- [ ] **Communication Plan**: Draft and approve the communication plan for the upgrade.
+
+## Pre-Upgrade
+
+- [ ] **Schedule Upgrade Window**: Decide on and communicate the upgrade window to all stakeholders.
+- [ ] **Notify Stakeholders**: Distribute the communication plan through all relevant channels.
+- [ ] **Prepare Upgrade Guide**: Ensure detailed upgrade instructions are available and easy to follow.
+- [ ] **Validator Coordination**: Confirm that validators are informed and prepared for the upgrade.
+
+## Execution
+
+- [ ] **Final Pre-Upgrade Check**: Conduct a last-minute check to ensure all systems and stakeholders are ready.
+- [ ] **Deploy Upgrade**: Initiate the runtime upgrade process according to the planned procedure.
+- [ ] **Monitor Deployment**: Closely monitor the network for any immediate issues during the upgrade.
```

Comment:
Shall we comment here about timezones, preferred time to do it or something like that?

Reply:
I like that, maybe use CET as a standard?

Reply:
Yep, let's use CET/CEST 👍🏻

Reply:
already updated